subsys/net/lib/lwm2m/lwm2m_object.h

```diff
@@ -147,6 +156,8 @@ struct lwm2m_message;
 #define LWM2M_PATH_LEVEL_RESOURCE 3
 #define LWM2M_PATH_LEVEL_RESOURCE_INST 4
 
+#define LWM2M_OBJLNK_NULL 65535 /* null link */
```

Comment:
There's `LWM2M_OBJLNK_MAX_ID` defined in `lwm2m.h`, I think it could be reused here (the spec says NULL objlnk should use the MAX_ID for both values).

Reply:
Ok, done.